docs/getting-started/quickstart-programming.md

```diff
@@ -3,29 +3,28 @@ sidebar_position: 2
 title: Programming Guide
 ---
 
-Let's learn how to build applications with [DBOS Transact](https://github.com/dbos-inc/dbos-ts).
-In this tutorial, we'll modify the example application from our [quickstart](./quickstart.md) to reliably send greetings to your friends.
-We'll show you how construct a reliable workflow that updates your database and calls third-party APIs.
+This tutorial assumes you have finished the [Quickstart](./quickstart.md) and you have a [DBOS Transact](https://github.com/dbos-inc/dbos-ts) app running locally. In this guide we'll modify that example to reliably record events across two different systems: Postgres and a third-party API. This app will write to both systems consistently, even if interrupted and restarted at any point.
 
-Before starting this tutorial, we recommend finishing the [quickstart](./quickstart.md).
-You can use the application from the quickstart to complete this tutorial.
+## Serving HTTP Requests
```

Comment:
I wonder if we should number these steps as we did in the quickstart? It seemed effective there.

Reply:
done